actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -90,18 +100,18 @@ class ActionCable.Connection
     open: ->
       ActionCable.log("WebSocket onopen event")
       @disconnected = false
```

Comment:
Reviewing the `open` handler and where it sets `@disconnected` is the only thing left to address before this can be merged, IMO.